.changeset/sixty-tips-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
+---
+
+Update `setTimeout()` to `queueMicrotask()`
```

Comment:
Maybe we should add a better explanation here.

Reply:
Done, let me know if it's fine, or modify at will :)